src/index.js

```diff
@@ -7,44 +7,47 @@ const {Command} = require('commander');
 minilog.enable();
 
 const program = new Command();
-
 program
     .option('-h, --host [address]', 'connection address')
     .option('-p, --port [number]', 'mqtt port')
     .option('-g, --game [filepath]', 'path to game');
-
 program.parse(process.argv);
-
 const inputs = program.opts();
     
 const virtualMachine = new VirtualMachine();
     
-if (process.title !== 'browser') {
-    console.log(inputs)
+if (process.title === 'browser') {
+
+    virtualMachine.start();
+
+} else {
+
+    virtualMachine.start();
     const host = inputs.host ? `${inputs.host}` : 'localhost';
     const port = inputs.port ? `${inputs.port}` : '1883';
     const gamePath = inputs.game ? `${inputs.game}` : `${process.cwd()}/game/DefaultGame.sb3`;
-
+    // params: (extensionId, peripheralId/connection address, port, userName, password)
+    virtualMachine.connectMqtt('playspot', host, port, '', '');
```

Comment:
do we want to check and make sure the connect succeeded before we load the game?